src/chainparams.cpp

```diff
@@ -218,11 +215,8 @@ class CTestNetParams : public CChainParams {
 
         // The miner fund is enabled by default on testnet.
         consensus.enableMinerFund = ENABLE_MINER_FUND;
-        consensus.payoutAddressSets = RewardAddresses::TestNet::AddressSets;
```

Comment:
Would prefer this wasn't renamed. Miner fund is kind of presuming miners are paying these addresses. They are not. But I won't block the PR over it. Just don't want future social drama based on misunderstandings.